src/mon/Monitor.cc

```diff
@@ -5937,6 +5938,19 @@ void Monitor::tick()
 
   mgr_client.update_daemon_health(get_health_metrics());
   new_tick();
+  // MON Memory Usage Logging:
+  // Capture and log memory metrics using MemoryModel.
+  // Establish a baseline for comparison.
+  static MemoryModel mm(g_ceph_context);
+  static MemoryModel::snap last;
+  mm.sample(&last);
+  static MemoryModel::snap baseline = last;
```

Comment:
This does appear to identical to the existing code in MDCache, but there are a few things I don't understand:
1. Won't last and baseline always match?
2. Why do mm, last, and baseline need to be static?  I'm not really ok with making these static without a specific reason.

It seems like mm and last can simply be local variables.  I presume that the intent is for baseline to contain the last recorded value so that the log line can contain the last value and the current value, so baseline should be a member of Monitor.